introduction.md

```diff
@@ -1 +1 @@
-Here's some stuff about myself. JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?